[Fix] Spring CI.yml 에 MySQL 관련 설정 추

❌ 버그 발생
CI/CD 파이프라인에서 ApplicationTests > contextLoads() 테스트가 실패하고 있습니다. 이는 Spring Bean 의존성 관련 문제로 인해 발생하는 것으로 보입니다.
![image](https://github.com/user-attachments/assets/84a0f248-fb72-49d0-bf66-fc65025f8938)


🐞 발생한 위치
위치: ApplicationTests > contextLoads()
파일: DefaultCacheAwareContextLoaderDelegate.java:180
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException at DefaultListableBeanFactory.java:1880

📖 참고 사항
에러 로그에 따르면, Spring의 Bean 초기화 과정에서 의존성을 주입하지 못해 발생한 문제입니다.
DB 연동 문제일 가능성이 있습니다.
이를 해결하기 위해 MySQL 설정을 추가할 예정입니다.
추가적으로 필요한 정보는 댓글로 남겨주세요.

--- a/.github/workflows/Spring CI.yml
+++ b/.github/workflows/Spring CI.yml
@@ -27,6 +27,15 @@ jobs:
         with:
           java-version: '21'
           distribution: 'temurin'
+      - name: Setup MySQL
+        uses: samin/mysql-action@v1
+        with:
+          host: ${{ secrets.RDS_HOST }}
+          port: ${{ secrets.RDS_PORT }}
+          database: ${{ secrets.RDS_NAME }}
+          user: ${{ secrets.RDS_USER }}
+          password: ${{ secrets.RDS_PASSWORD }}
+          character set server: 'utf8'
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew

